recipes/decorator/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "4.0.9" %}
+
+package:
+  name: python-decorator
```

Comment:
Package name didn't match directory name. This was tracked under issue ( https://github.com/conda-forge/staged-recipes/issues/720 ) and corrected in PR ( https://github.com/conda-forge/staged-recipes/pull/724 ).

This is yet another occurrence of issue ( https://github.com/conda-forge/staged-recipes/issues/1073 ).